product/views/StorageResource.yaml

```
@@ -43,14 +43,21 @@ col_order:
 - logical_free
 - logical_total
 
+col_formats:
+  -
+  -
+  -
+  - :bytes_human
+  - :bytes_human
```

For consistency, let's left-align the array

```suggestion
-
-
-
- :bytes_human
- :bytes_human
```

---

done